utils/interdomain/interdomainutils.go

```diff
@@ -0,0 +1,26 @@
+package interdomainutils
```

Comment:
Interdomain is just another feature in the list of NSM capabilites. Let's nto distiguish it fromt he rest of the code. Please merge this in the generic "utils".

Reply:
Hello @nickolaev. Proxy NSMD is the main part of Interdomain NSM and designed specifically for this. If we present current Proxy NSMD without another interdomain changes, it will be just useless feature without usage.